docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -0,0 +1,871 @@
+
+
+[float]
+[[logstash-setup]]
+==== Configure {ls} to read from the {beats} input
+
+{ls} provides {logstash-ref}/input-plugins.html[input plugins] for reading from
+a variety of inputs. In this tutorial, you create a {ls} pipeline configuration
+that reads from the {beats} input and sends events to the {es} output.
+
+To configure {ls}:
+
+. Create a new {ls} pipeline configuration file called `logstash.conf` that
+contains:
++
+--
+* An input stage that configures {ls} to listen on port 5044 for incoming {beats}
+connections.
+* An output stage that indexes events into {es}. The output stage also
+configures {ls} to write to the {metricbeat} index.
+--
++
+For example:
++
+[source,ruby]
+----
+input {
+  beats {
+    port => 5044
+  }
+}
+
+# The filter part of this file is commented out to indicate that it
+# is optional.
+# filter {
+#
+# }
+
+output {
+  elasticsearch {
+    hosts => "localhost:9200"
+    manage_template => false
+    index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}" <1>
+  }
+}
+----
++
+This configuration allows you to route events to {ls}, where you have full
+access the {ls} capabilities for collecting, enriching, and transforming
+data.
+
+
+[float]
+==== Start {ls}
+
+Use the command that works with your system. If you installed {ls} as a deb or
+rpm package, make sure the config file is in the `config` directory.
+
+*deb:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+sudo /etc/init.d/logstash start
+----------------------------------------------------------------------
+
+*rpm:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+sudo service logstash start
+----------------------------------------------------------------------
+
+*mac:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+./bin/logstash -f logstash.conf
+----------------------------------------------------------------------
+
+*win:*
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+bin\logstash.bat -f logstash.conf
+----------------------------------------------------------------------
+
+NOTE: The default configuration for {beats} and {ls} uses plain TCP. For
+encryption you must explicitly enable SSL when you configure {beats} and {ls}.
+
+{ls} reads from the {beats} input and indexes events into {es}. You haven't
+defined a filter section yet, so {ls} simply forwards events to {es} without
+additional processing. Next, you learn how to define the filter stage.
+
+[float]
+[[logstash-filter]]
+==== Define a filter to extract data from a field
+
+The system metrics collected by {metricbeat} include a field called `cmdline`
+that contains the full command-line argument used to start the process. For
```

Comment:
I don't know Logstash or Metricbeat very well at all, so it wasn't clear to me what "the process" was here.  I'm presuming that there are a specific set of processes that Metricbeat collects info about, but it might be nice to be clearer if possible.

Reply:
Yah I should beef up the intro with that info actually!